core/src/main/scala/kafka/network/RequestChannel.scala

```diff
@@ -207,6 +286,28 @@ object RequestChannel extends Logging {
       math.max(apiLocalCompleteTimeNanos - requestDequeueTimeNanos, 0L)
     }
 
+    def getConsumerFetchSizeBucketMetricName: Seq[String] = {
```

Comment:
I feel returning Option[String] would make it easier to understand.
Later on, when a Seq is needed, it can be converted with the "Option.toSeq" method.

Reply:
Using Seq is to make the caller method more concise. If it returns Option, the caller method updateRequestMetrics need to check whether it is None. If it is None, do nothing, if it is Some(String), append. Since this is a helper method, I feel it is better to return Seq such that the caller could use the result directly without checking the result.

Reply:
It seem working to call Seq ++ Option directly. Is this a good code style?

Reply:
I mean if we return an Option[String] here, it can be easily turned into an Seq[String] with the ".toSeq" method,
e.g.
`
val metricName = getConsumerFetchSizeBucketMetricName() // the type of metricName will be Option[String]
`
Later one in the expression we can do
`
... ++ metricName.toSeq ++ ...
`
that way the resultant Seq[String] will be Seq.empty if the returned value is None or a non-empty Seq if the returned value is Some.

The reason for raising this comment is that the test code is showing 
"getConsumerFetchSizeBucketMetricName(0)", which is a bit strange since this method can return at most 1 effective string.

Reply:
got it. updated as suggested